web/src/components/ExpandCollapseString.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactElement } from "react";
+import ClampLines from "react-clamp-lines";
+
+interface Props {
+  text: string;
+  id: string;
+  dataTestId?: string;
+  viewMoreText: string;
+  viewLessText: string;
+}
+
+export const ExpandCollapseString = (props: Props): ReactElement => {
+  return (
+    <div {...(props.dataTestId ? { "data-testid": props.dataTestId } : {})}>
+      <ClampLines
+        text={props.text}
+        id={props.id}
+        lines={2}
```

Comment:
In order for this to be more reusable, should the number of lines be a prop maybe?

Reply:
I was debating whether they should be a prop now or in the future when the need arose. I can make the update now.

Reply:
Yeah, I'm usually opposed to extra props until the need arises, totally. I just thought maybe in this case, it was important enough to the usage of the component that it seemed like a good candidate